pages/reference/hardware/versioning.md

```diff
@@ -0,0 +1,56 @@
+
+As an example the [Raspberry Pi 1 model B+](https://www.raspberrypi.org/products/raspberry-pi-1-model-b-plus/) and [Raspberry Pi Zero W](https://www.raspberrypi.org/products/raspberry-pi-zero-w/) are part of the same device type since they are both `armv6l` architecture SoCs and the boot loader is capable of booting both boards. However, the [Raspberry Pi 3 model B+](https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/) is by default a `armv7l` CPU and we therefore separate it into a new device type.
+
+In balenaCloud the device type is also used to define what architecture the built containers for each app will be. So in the above example RPi ZERO deployments are built for the `armv6l` architecture where as RPi 3 applications will target the `armv7l` architecture. 
+
+### Designations
+
+When looking at all the supported device types in balenaCloud, you will notice that there are a number of different designations. Here we will discuss a few of these designations in more detail.
+
+#### Official
+All device types that are not designated with `COMMUNITY` or `PRIVATE` are considered officially supported boards. 
+
+Official device types are:
+- Run through a full suite of testing on real hardware each time an OS version is released for the device type.
+- We endevour to ensure that all the hardware peripherals on the device type are functional and accessible. This includes things like i2c, SPI, audio, etc.
+- Continually updated to new versions of the OS, usually within 2 or 3 minor versions of meta-balena master version.
+- Official devices are recommended for production deployments.
+
+On some official device types you will see the `(BETA)` designation. This does not mean the OS itself is `beta` level software, but rather that this device type not yet used by enough people or has not been in production for a long enough time. In balenaCloud all official device types start out as `BETA` and when the team decides the device type has enough production usage, it is graduated out of `BETA`.
+
+#### Private
+Private device types have exactly the same guarentees and testing process as official device types but they are only visible to specific customers that have paid for this device support. If you are interested in finding out more about private device support email solutions@balena.io .
+
+#### Community
+Device types that are marked as `COMMUNITY` are device types that have been contributed by an external community member rather than the balena team. An example of a community device type is the Orange Pi Zero which was contributed into the https://github.com/balena-os/balena-allwinner repo.
+
+The community devices types are:
+- Maintained by the community member
```

Comment:
would be useful to add where the community member should speak up like that? I'm guessing in the forums?